src/main/resources/assets/computercraft/lua/rom/programs/shell

```diff
@@ -180,7 +180,11 @@ function shell.programs( _bIncludeHidden )
 	-- Sort and return
 	local tItemList = {}
 	for sItem, b in pairs( tItems ) do
-		table.insert( tItemList, sItem )
+	    if sItem:find(".lua") == sItem:len()-3 then
```

Comment:
What's the best code, to stop inserting items twice? I would use this code:
local tItemList = {}
local checkItem = {}
	for sItem, b in pairs( tItems ) do
	    if sItem:find(".lua") == sItem:len()-3 then
               if checkItem[sItem] ~= true then
	        table.insert( tItemList, sItem:sub(1,-5) )
                checkItem[sItem:sub(1,-5)] = true
             end
            else
                if checkItem[sItem] ~= true then
	          table.insert( tItemList, sItem )
                   checkItem[sItem] = true
                end
	    end
	end
	table.sort( tItemList )
return tItemList
Just for show and not tested yet. Is there a better solution than mine?